### ISSUE TITLE
Reordering ignore namespace prefixes

### ISSUE THREAD

**Issue by jur007** [Xavalon/XamlStyler#111]
Hi grochocki,

I have attached properties at controls like "common:MarginHelper.AutoMargin="True""
and enabled the "OrderAttributesByName".

Is it possible to order this attribute as "M" and not as "c"? Maybe an option to exclude prefixes?

**Comment by grochocki** [comment]
After reviewing this based on our [feature request guidelines](https://github.com/Xavalon/XamlStyler/blob/master/CONTRIBUTING.md#guidelines-for-feature-requests), we feel like this particular request reflects a personal style preference that might not be a good fit for the larger developer community, so we are declining this feature suggestion. Thanks for your contribution!

**Comment by United600** [comment]
Hi.
I think, after #306 merging, this should be reconsidered.
IIRC [conditional XAML](https://learn.microsoft.com/en-us/windows/uwp/debug-test-perf/conditional-xaml) is gone in WinUI3 but it's still moderately used in UWP, so let's take a look at an example and current pain point:

#### Current result (with default config)
``` xml
<Button
    x:Name="Example"
    Height="100"
    Margin="4,0,0,0"
    VerticalAlignment="Center"
    HorizontalAlignment="Center"
    contract5NotPresent:Width="52"
    contract5Present:Width="64"
    Content="Button" />

}
```

#### Desired result
``` xml
<Button
    x:Name="Example"
    contract5NotPresent:Width="52"
    contract5Present:Width="64"
    Height="100"
    Margin="4,0,0,0"
    VerticalAlignment="Center"
    HorizontalAlignment="Center"
    Content="Button" />
}
```

My propose would be to extend the 'IgnoreDesignTimeReferencePrefix' to support a customizable list of AttachedProperties, so it doesn't separate rule groups.

``` json
{
    "IgnoreAttachedPropertiesPrefix": "d:*, contract*:*"
}
```

Thanks.

**Comment by United600** [comment]
@grochocki Sorry for the ping. I didn't create a new issue, since this closed one pretty much covers it.
